docs/detailed-result-handling.md

```diff
@@ -20,17 +20,28 @@ simple methods available that you can access anywhere.
 * `isSuccess()`: Simply states whether or not the search was a success, or error.
 * `getRecords()`: A `PaginatedList` of `Record` objects that were returned by the search service based on your `Query`.
 * `getFacets`: An `ArrayList` of `Facet` objects that were returned by the search service based on your `Query`.
-* `getQuery`": The `Query` object
```

Comment:
one last thing - I think we can still document the method exists maybe just refer/link to the docs further down